### PR TITLE
Make access token private

### PR DESCRIPTION
**Pull request by urielsalis**
## Purpose
When a access token is detected in a attachment, make the ticket private

## Approach
Check for "--accessToken ey"

### CODE REVIEW

**Review by SPGoding — DISMISSED** [review]

**Review by SPGoding — APPROVED** [review]

File: src/test/kotlin/io/github/mojira/arisa/modules/PrivacyModuleTest.kt [hunk 1]
```diff
@@ -252,6 +252,24 @@ class PrivacyModuleTest : StringSpec({
...
         hasSetPrivate shouldBe true
     }
+    
+     "should mark as private when the attachment contains access token" {
+        var hasSetPrivate = false
+
+        val issue = mockIssue(
+            attachments = listOf(
+                mockAttachment(
+                    getContent = { "--uuid 1312dkkdk2kdart342 -accessToken eyJimfake.12345.fakestuff --userType mojang".toByteArray() }
```
Comment (SPGoding) [inline src/test/kotlin/io/github/mojira/arisa/modules/PrivacyModuleTest.kt:262]:
```suggestion
                    getContent = { "--uuid 1312dkkdk2kdart342 --accessToken eyJimfake.12345.fakestuff --userType mojang".toByteArray() }
```

File: src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt [hunk 2]
```diff
@@ -12,7 +12,8 @@ class PrivacyModule(
 ) : Module {
     private val patterns: List<Regex> = listOf(
         """\(Session ID is token:""".toRegex(),
-        """(?<!\[~)\b[a-zA-Z0-9.-_]+@[a-zA-Z0-9.-_]+\.[a-zA-Z0-9.-_]{2,}\b(?!])""".toRegex()
+        """(?<!\[~)\b[a-zA-Z0-9.-_]+@[a-zA-Z0-9.-_]+\.[a-zA-Z0-9.-_]{2,}\b(?!])""".toRegex(),
+        """--accessToken ey""".toRegex()
```
Comment (violine1101) [inline src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt:16]: Does the access token always start with `ey`? Isn't it possible that it starts with something else?

Comment (urielsalis) [reply src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt:16]: Its a JWT token. They are 3 base64-encoded jsons. ey translates to { which must be the first character of the header(first part)

Comment (violine1101) [reply src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt:16]: Ah yeah, makes sense 👍

**Review by SPGoding — APPROVED** [review]

**Review by violine1101 — APPROVED** [review]

**Review by chandler05 — APPROVED** [review]

**Pull request merged**